packages/connect-common/package.json

```diff
@@ -19,7 +19,6 @@
     "main": "./lib/index.js",
     "files": [
         "lib/",
-        "!lib/storage",
```

Comment:
Should it have been published since it was moved from the Connect repository?

Reply:
imho needs to be published to resolve this https://github.com/trezor/trezor-suite/issues/5358#issuecomment-1161819401